spec/models/presentation_spec.rb

```diff
@@ -10,11 +10,15 @@
 
     it { is_expected.to validate_presence_of :topic }
     it { is_expected.to validate_presence_of :duration }
-    it { is_expected.to validate_numericality_of :duration }
 
     it { is_expected.to belong_to :person }
     it { is_expected.to belong_to :event }
 
+    it do
```

Comment:
If youre going to use the multiline it statement you should probably add a text description and use an explicit subject. 

it might just be a convention thing... but its a convention that I like a lot